README.md

```diff
@@ -48,6 +48,9 @@
 
```

Comment:
**[OPTIONAL]**

- [ ] Kindly follow Gitflow because in this project, you were supposed to use Gitflow means you will make your dev branch default and then you will make only one single branch from the dev branch where you will make all your app and submit that one single branch for a review and after getting approval you were supposed to merge that one single branch into dev branch :thinking: Kindly use this approach next tie while doing a Solo Capstone Project :+1: The approach which you have adopted is applicable in Group Capstone projects :blush:

Reply:
Well noted @AmaduKamara . Thanks.